tests/roles/ovn_adoption/tasks/main.yaml

```diff
@@ -29,31 +29,77 @@
   retries: 60
   delay: 2
 
-- name: get podified OVN NB ovsdb-server IP
+- name: get podified OVN NB ovsdb-server service cluster IP
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc get pod ovsdbserver-nb-0 -o jsonpath='{.metadata.annotations.k8s\.v1\.cni\.cncf\.io/network-status}' | \
-    jq 'map(. | select(.name=="openstack/internalapi"))[0].ips[0]' | \
-    tr -d '"'
+    oc get svc --selector "statefulset.kubernetes.io/pod-name=ovsdbserver-nb-0" -ojsonpath='{.items[0].spec.clusterIP}'
   register: podified_ovn_nb_ip_result
 
 - name: get podified OVN SB ovsdb-server IP
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc get pod ovsdbserver-sb-0 -o jsonpath='{.metadata.annotations.k8s\.v1\.cni\.cncf\.io/network-status}' | \
-    jq 'map(. | select(.name=="openstack/internalapi"))[0].ips[0]' | \
-    tr -d '"'
+    oc get svc --selector "statefulset.kubernetes.io/pod-name=ovsdbserver-sb-0" -ojsonpath='{.items[0].spec.clusterIP}'
   register: podified_ovn_sb_ip_result
 
+- name: set fact for oc run args
```

Comment:
This and the relevant `oc exec` part is not reflected in the docs. We should keep the parity, ideally with verbatim copy/paste (exact same content in docs and tests as much as possible -- using shell vars for parametrization where necessary).

I will also bring this up to others to make sure we're comfortable with the `oc exec` approach in general. This is a bit double bladed, it does make the manual workflow more complex, but the procedure also becomes better insulated from possible quirks of the "users laptop" where the procedure is being executed, so i'm leaning +1.

Reply:
+1

Reply:
but I'm updating other places now, let's keep it synced, right